CONTRIBUTING.md

```diff
@@ -42,5 +49,41 @@ Included in this repo is the file `helphub.wordpress.2017-06-15.xml`. Import thi
+
+#### Helphub Theme
+
+**Requirements:**
+
+- [npm](https://www.npmjs.com/get-npm)
+- [gulp](https://gulpjs.com/)
+- [SASS](http://sass-lang.com/)
+
+Helphub theme uses [SASS](http://sass-lang.com/) for applying styles as it provides possibility for breaking one large style.css file into smaller partials and, therefore, reduce possible GIT conflicts caused with multiple modifications of the same file in different branches. Maintenance is, also, greatly improved. Once modified SASS (.scss) files need to be compiled into style.css. Compiler of our choice is [gulp](https://gulpjs.com/). To be able to run gulp you need [npm](https://www.npmjs.com/get-npm). We will assume that you already have npm.
+
+In terminal navigate to theme folder. First we need to install dependencies from `package.js` file:
+```
+npm install
+```
+
+After this command has ran in your terminal you'll have another folder in theme root, `node_modules`. This folder is ignored in .gitignore and contains all the tools we need for running gulp tasks, defined in `gulpfile.js`. This file contains automated tasks for the project (which can be anything - building svg sprites, minifying css and js files etc). We are using it for two tasks: `css` and `watch`.
+
+`css` task compiles all .scss files into `style.css`. This means that every time you run this task `style.css` will be overridden with new code from .scss files, located in `sass` folder. Hence, instead modifying style.css directly, all css changes are to be added into appropriate .scss partial after which you should run compiler in order to see your changes. Compiling is done with running following command in terminal:
+
+```
+gulp css
+```
+
+If, however, you are adding a lot of css changes, instead of running a lot of `css` tasks, you can run `watch` task, like so:
+
+```
+gulp watch
+```
+
+This task tells compiler to watch all changes created in .scss files and rebuild new style.css every time you save the file.
+
+While you can use regular css syntax in .scss files (as long as Travis tests are passed), we would like to encourage you to [learn](http://sass-lang.com/guide) and use SASS as much as possible. It's good for you and for project.
+
 ### Propose design suggestions and improvements
-Join our discussions in the [#docs](https://make.wordpress.org/docs/tag/docs/) slack channel on **Thursdays from 17:00 - 18:00 UTC** and offer your help. Or submit your ideas on our Github repo here: https://github.com/Kenshino/HelpHub/issues. 
+
+Join our discussions in the [#docs](https://make.wordpress.org/docs/tag/docs/) slack channel on **Thursdays from 17:00 - 18:00 UTC** and offer your help. Or submit your ideas on our Github repo here: https://github.com/Kenshino/HelpHub/issues.
```

Comment:
Yup, looks like it, we should also link `slack` to make.wordpress.org/chat as well I'd say.